gradle.properties

```diff
@@ -1,21 +1,14 @@
 kotlin.code.style=official
-kotlin_version = 1.4.20
 kotlin.mpp.enableGranularSourceSetsMetadata=true
 kotlin.native.enableDependencyPropagation=false
-serialization_version=1.0.1
-mockk_version=1.10.3-jdk8
-lifecycle_version=2.2.0
-trikot_foundation_version=1.0.21
-trikot_analytics_version=1.0.2
-trikot_streams_version=1.0.8
-trikot_viewmodels_version=1.0.6
-trikot_http_version=1.0.7
-trikot_kword_version=1.0.2
-trikot_streams_android_ktx_version=1.0.2
-trikot_viewmodels_android_ktx_version=1.0.4
-trikot_http_android_ktx_version=1.0.2
-trikot_kword_android_ktx_version=1.0.1
-trikot_framework_name=TrikotFrameworkName
+
+trikot_foundation_version=2.1.0
```

Comment:
Any reason to not use the same pattern has others within the `Versions.kt`?

Reply:
Yeah, since we share the same version number from the gradle.properties file with the podspec, we had to keep them here. Otherwise it'd have been a bit more complicated to load kotlin version numbers from the podspec.